apps/vr-tests-react-components/.storybook/preview.js

```diff
@@ -26,27 +26,21 @@ setAddon({
    */
   addStory(storyName, storyFn, config = {}) {
     this.add(storyName, context => {
-      return <FluentProvider theme={webLightTheme}>{storyFn(context)}</FluentProvider>;
+      return (
+        <>
+          <FluentProvider theme={webLightTheme}>{storyFn(context)}</FluentProvider>;
+          {config.includeDarkMode && <FluentProvider theme={webDarkTheme}>{storyFn(context)}</FluentProvider>}
```

Comment:
**suggestion:**

we could add it to a dictionary map and render  via map. WDYT?

root scope:

```js
const configToPropsMap = {
  includeDefault: { theme: webLightTheme },
  includeDarkMode: { theme: webDarkTheme },
  includeHighContrast: { theme: webHighContrastTheme },
  includeRtl: { theme: webLightTheme, dir: 'rtl' },
};

const configDefaults = { includeDefault: true };
```

addStory:

```js
addStory(storyName, storyFn, config = {}) {
    const modeConfig = { ...configDefaults, config };
    this.add(storyName, context => {
      return (
        <>
          {Object.keys(modeConfig).map(configProp => {
            return (
              <FluentProvider key={configProp} {...configToPropsMap[configProp]}>
                {storyFn(context)}
              </FluentProvider>
            );
          })}
        </>
      );
    });

    return this;
  },
```

types extenstion:

```ts
export interface AddStoryConfig {
  /**
   * @default true
   */
  includeDefault?: boolean;
}
```